src/Features/Core/Portable/Completion/CompletionListProvider.cs

```diff
@@ -11,10 +13,24 @@ namespace Microsoft.CodeAnalysis.Completion
     /// </summary>
     internal abstract class CompletionListProvider
     {
+
+        public async Task ProduceCompletionListAsync(CompletionListContext context)
```

Comment:
I guess I was imagining we'd detect this specific situation in the _caller_ to `ProduceCompilationListAsync`, and then non-fatal in that particular case.